pb/terrarium/provider/provider.proto

```diff
@@ -84,11 +84,11 @@ message ProviderRequest {
   string arch = 4;
 }
 
-message UploadProviderSourceZipRequest {
+message UploadProviderBinaryRequest {
   ProviderRequest provider = 1;
   bytes zip_data_chunk = 2;
```

Comment:
does it needs to be a zip ? or will it be a raw binary ?

Reply:
It needs to be a zip.